CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# Prerequisite Skills to Contribute
+- [Git](https://git-scm.com/) 
+- [node.js](https://nodejs.org/en/)
```

Comment:
I believe we are not using nodejs for website currently, instead add React to it, javascript maybe 
cc @verma-kunal @pavangudiwada